src/components/PaginationButtons.js

```diff
@@ -0,0 +1,59 @@
+import '../styles/pageButton.css';
+
+function PageButton({ children, onClick, isActive, isLoading }) {
+  return (
+    <button
+      onClick={onClick}
+      className={`page-button ${isActive ? 'page-button__active' : ''}`}
+      disabled={isLoading}
+    >
+      {children}
+    </button>
+  );
+}
+
+export default function PaginationButtons({
+  onClick,
+  page,
+  setPage,
+  isLoading,
+}) {
+  const onPreviousPage = () => {
+    const isPageOutOfBounds = page <= 1;
+    isPageOutOfBounds || onClick(page - 1);
+  };
+
+  const onPagination = (nextPage) => {
+    onClick(nextPage);
+  };
+
+  const onNextPage = () => {
+    const isPageOutOfBounds = page >= 5;
+    isPageOutOfBounds || onClick(page + 1);
+  };
+
+  return (
+    <div className="pagination-buttons">
+      <PageButton onClick={onPreviousPage} isLoading={isLoading}>
+        &lt;
```

Comment:
html이 아닌 jsx 문법이므로 string 값으로 넣어주셔도 무방한 부분입니다.
```suggestion
       {'<'}
```